### PR TITLE
scheduler_tool: define prci_config as part of nightly jobs definition

### PR DESCRIPTION
**Pull request by f-trivino**
prci_definition defines the set of tests to be executed. It is different for each
nightly job, hence this commit defines the proper prci_definition per nightly flow.
In addition, openclose_pr folder is deleted as it is no longer used.

### CODE REVIEW

**Comment by dhnunes** [comment]
Hi @f-trivino 
I agree with Flo regarding to doc change too.

Since we are about to implement versioning in PR-CI, please submit the requested changes and we can dd in version 1.0.0

File: ansible/roles/automation/nightly_pr/tasks/main.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
            --repo_path {{ basedir }}/freeipa
            --flow {{ item.flow }}
            --branch {{ item.branch }}
-           --prci_config {{ prci_config }}
+           --prci_config ipatests/prci_definitions/{{ item.prci_config }}
```
Comment (flo-renaud) [inline ansible/roles/automation/nightly_pr/tasks/main.yml]: `ipatests/prci_definitions` can be replaced with `{{ prci_def_dir }}`.

File: doc/prscheduler.md [hunk 2]
```diff
@@ -41,22 +41,22 @@ of the script on a scheduler machine
...
 
-This script is automating the creation of a dummy pull request against
-the official freeipa upstream repository. It is launched on a machine called
-the scheduler. This machine does not need to be very beefy, as it only
-creates a pull request (the tests are executed on PR-CI runners when they
-detect that a new PR has been created).
+This script currently supports two main commands: open_nightly_pr and
+open_template_pr.
+
+Open_nightly_pr command simply automate the creation of a pull request
```
Comment (flo-renaud) [inline doc/prscheduler.md]: typo: automate**s**

File: doc/prscheduler.md [hunk 3]
```diff
@@ -145,19 +148,19 @@ ansible-playbook -i ansible/hosts/runners ansible/prepare_openclose_pr_tool.yml
 The playbook will prompt you for variables:
 ```
 # ansible-playbook -i ansible/hosts/runners ansible/prepare_openclose_pr_tool.yml
-Provide an unique identifier for the PR: testing_master    # $identifier
 Repo owner to create branches: freeipa-pr-ci               # $repo_owner
 GitHub token: 123qwerty                                    # $github_token
 Provide the path to a test definition file to use it to open new PRs. For example: ipatests/prci_definition/gating	# $prci_config, for instance ipatests/prci_definition/nightly_master.yaml
```
Comment (flo-renaud) [inline doc/prscheduler.md]: Here we provide only the name of the file (the path is set in $prci_def_dir).

File: ansible/roles/automation/nightly_pr/tasks/main.yml (hunk 1 above)
Comment (f-trivino) [reply ansible/roles/automation/nightly_pr/tasks/main.yml]: much better like that, thanks.

File: doc/prscheduler.md (hunk 2 above)
Comment (f-trivino) [reply doc/prscheduler.md]: done!, thanks.

File: doc/prscheduler.md (hunk 3 above)
Comment (f-trivino) [reply doc/prscheduler.md]: I thought I amended this part of the doc accordingly.... It is completed now.

**Pull request merged**